creator.cpp

```diff
@@ -231,7 +231,7 @@ bool Creator::showRootMessageBox()
     msgBox.setText(tr("Root privileges required to write image.\nRun application with sudo."));
     msgBox.setIcon(QMessageBox::Critical);
     msgBox.setStandardButtons(QMessageBox::Ok);
-    msgBox.setButtonText(QMessageBox::Ok, tr("OK"));
+    msgBox.addButton(QMessageBox::Ok);
```

Comment:
I think you can drop this line completely since you already use `setStandardButtons`, in other places as well

Reply:
Done